docs/en-US/adding-a-new-site/changing-php-version.md

```diff
@@ -52,15 +52,15 @@ extensions:
     - webgrind
     - trusted-hosts
     - tls-ca
-    - php56
     - php71
+    - php74
 ```
 
-## Forcing a Version of PHP
+## Forcing a Version of PHP for Nginx
```

Comment:
This isn't true if we're using `php: 8.0` in the config, because `php:` is a full replacement of `nginx_upstream`. This entire doc may need rewriting to focus exclusively on the new `php:` option with a minor note about `nginx_upstream` for completeness